include/iceflow/consumer-tlv.hpp

```diff
@@ -193,23 +221,30 @@ class ConsumerTlv {
           m_names[interestUri].push_back(manifestName);
         }
       } break;
+        // JSON carries the analyzed results of the compute function of the
+        // upstreams.
       case Json: {
         std::vector<std::string> strs;
         boost::split(strs, interest.getName().toUri(), boost::is_any_of("/"));
+
+        // Pushing the Json Data to the input queue of the consumer
         addBlockToInputQueue(Block(data.getContent(), data.getContentType()));
         // need to update a manifest of json names and not only one data item
         /////////////////////////////////////////////////
-        // stream
+        // stream number
         int manifestStreamCount = stoi(strs[strs.size() - 2]);
         // data sequence
         int dataCount = stoi(strs[strs.size() - 1]);
         // key is the manifest id a data object belongs to
+        // whenever this case has been triggered then the keyy value is getting
+        // 0. Not getting why wit is here
         int key = 0;
```

Comment:
It's best to add such comments to commits or PRs, I would say.

Regarding the actual variable name, I think we renamed it to `lowestSequenceNumber`, right?

```suggestion
        int lowestSequenceNumber = 0;
```